src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt

```diff
@@ -96,11 +96,9 @@ fun insertTitleAndResizableTextIntoPanelColumns(
 
 fun snykCodeAvailabilityPostfix(): String {
     val sastOnServerEnabled = pluginSettings().sastOnServerEnabled
-    val localCodeEngineEnabled = pluginSettings().localCodeEngineEnabled
     return when {
         !isSnykCodeAvailable(pluginSettings().customEndpointUrl) -> " (disabled for endpoint)"
-        (sastOnServerEnabled == null && localCodeEngineEnabled == null) -> " (unreachable server settings)"
-        (sastOnServerEnabled != false && localCodeEngineEnabled == true) -> " (disabled due to Local Code Engine)"
+        sastOnServerEnabled == null -> " (unreachable server settings)"
```

Comment:
Couldn't this also be a misconfigured URL? Is this message still correct?

Reply:
That's correct, this could also be a 422 response from the server. 

We have two options:

1. add `sastResponseError:boolean` to `pluginSettings()`, then use this logic here
2. reword `unreachable server settings` to more generic, like `something went wrong while fetching SAST seetings`.

from UX perspective, option 1 is better. What do you think?

Reply:
I'd go for option 1.

Reply:
Done